extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -1162,11 +1183,11 @@ public static function getCurrentStagingHost( $dbName='', $default='', $host = n
 		}
 
 		if ( preg_match( '/^(demo-[a-z0-9]+)-[s|r][0-9]+$/i', $host, $m ) ) {
-			return $m[ 1 ] . '.' . ( $dbName ? $dbName : 'www' ) . '.wikia.com';
+			return $m[ 1 ] . '.' . ( $dbName ? $dbName : 'www' ) . self::WIKIA_COM;
 		}
 
 		if ( in_array( $host, $wgStagingList ) ) {
-			return $host . '.' . ( $dbName ? $dbName : 'www' ) . '.wikia.com';
+			return $host . '.' . ( $dbName ? $dbName : 'www' ) . self::WIKIA_COM;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)